gssapi/raw/ext_ggf.pyx

```diff
@@ -1,3 +1,16 @@
+"""
+GGF Extensions
+
+GGF provides extended credential and security context inquiry that allows
+application to retrieve more information about the client's credentials and
+security context. One common use case is to use gss_inquire_sec_context_by_oid
+to retrieve the "session" key that is required by the SMB protocol for signing
+and encrypting a message. These calls are provided as a part of the raw
+interface and are not exposed in the high-level interface.
```

Comment:
no need for the sentence starting with "These calls".  It just means if we ever expose it at the high level, we'll have to remember to remove that sentence.